evm/v0.5/package.json

```diff
@@ -17,7 +17,7 @@
     "prepublishOnly": "yarn setup && yarn lint && yarn test",
     "test": "yarn test:v1 && yarn test:v2",
     "test:v1": "tsc && truffle test ./dist/test/*_test.js",
-    "test:v2": "jest --testTimeout 80000",
+    "test:v2": "jest --testTimeout 80000 --detectOpenHandles --forceExit",
```

Comment:
Could you explain why this has been added in a comment?

Reply:
Yeah, its because the either the sol artifact adapter or ethereum subprovider isnt releasing a resource on exit. Ill add a comment and re-tag you